src/node_sqlite.cc

```diff
@@ -930,7 +930,8 @@ bool StatementSync::BindParams(const FunctionCallbackInfo<Value>& args) {
   int anon_idx = 1;
   int anon_start = 0;
 
-  if (args[0]->IsObject() && !args[0]->IsUint8Array()) {
+  if (args[0]->IsObject() && !args[0]->IsUint8Array() &&
+      !args[0]->IsTypedArray() && !args[0]->IsArrayBuffer()) {
```

Comment:
Using `IsArrayBufferView()` here is likely a lot better than separate `IsUint8Array()` and `IsTypedArray()` checks.